agent/build.gradle.kts

```diff
@@ -26,104 +26,27 @@ dependencies {
   upstreamAgent("io.opentelemetry.javaagent:opentelemetry-javaagent")
 }
 
+licenseReport {
+  configurations = arrayOf(
+    project.configurations.bootstrapLibs.name,
+    project.configurations.javaagentLibs.name
+    // No need to include the upstreamAgent, because it already has all dependent licenses packaged
+    // Those will be preserved in our JAR
```

Comment:
Is is due to this change that we see so many differences in the `more-licences.md` file ? However the `licenses.md` remains perfectly identical with this PR.